onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1132,6 +1154,71 @@ def _should_skip_test_sample(op_name: str, sample) -> Optional[str]:
     return None
 
 
+class OrtAbortedError(RuntimeError):
+    """ONNX Runtime Aborted."""
+
+
+def _ort_session_run(serialized_model, ort_inputs, return_dict) -> None:
+    """Run a model with ONNX Runtime and store the results in return_dict."""
+
+    # Handle SIGSEGV so Python does not abort
+    def sig_handler(signum, frame):
+        warnings.warn(f"Segmentation error ({signum}) detected in ORT:\n{frame}")
+
+    signal.signal(signal.SIGSEGV, sig_handler)
+
+    # Disable all ORT optimizations
+    session_options = onnxruntime.SessionOptions()
+    session_options.graph_optimization_level = (
+        onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
+    )
+    try:
+        session = ort.InferenceSession(serialized_model, session_options)
+        results = session.run(None, ort_inputs)
```

Comment:
what happens when `SIGSEGV` is received?

Reply:
It will abort in this case.

Reply:
Would it behave differently without calling `signal.signal(signal.SIGSEGV, sig_handler)` (other than the warning)?

Reply:
The whole thing will then crash. It looks like it was propagated to the parent process?